### PR TITLE
Complete initial version of IggyClientV2

### PR DESCRIPTION
**Pull request by spetz**

### CODE REVIEW

File: sdk/src/clients/client_v2.rs [hunk 1]
```diff
@@ -0,0 +1,931 @@
...
+        strategy: &PollingStrategy,
+        count: u32,
+        on_message: Option<F>,
+        config_override: Option<PollMessagesConfig>,
+    ) -> JoinHandle<()>
+    where
+        F: Fn(PolledMessage) + Send + Sync + 'static,
+    {
+        let client = self.client.clone();
+        let mut interval = Duration::from_millis(100);
```
Comment (BartoszCiesla) [inline sdk/src/clients/client_v2.rs]: Is this 100 a default value which is the same as in `PollMessagesConfig` struct default trait implementation? Perhaps some internal const value could be used to make it explicit and use in both places?

File: sdk/src/clients/client_v2.rs [hunk 2]
```diff
@@ -0,0 +1,931 @@
...
+            loop {
+                sleep(interval).await;
+                let mut send_messages_batch = send_messages_batch.lock().await;
+                if send_messages_batch.commands.is_empty() {
+                    continue;
+                }
+
+                let mut initialized = false;
+                let mut stream_id = Identifier::numeric(1).unwrap();
+                let mut topic_id = Identifier::numeric(1).unwrap();
```
Comment (BartoszCiesla) [inline sdk/src/clients/client_v2.rs]: Use `default()`

File: sdk/src/clients/client_v2.rs (hunk 1 above)
Comment (spetz) [reply sdk/src/clients/client_v2.rs]: Fixed :)

File: sdk/src/clients/client_v2.rs (hunk 2 above)
Comment (spetz) [reply sdk/src/clients/client_v2.rs]: Fixed :)

**Review by BartoszCiesla — APPROVED** [review]

**Pull request merged**